### PR TITLE
Operation to repair file blob metadata

### PR DESCRIPTION
**Pull request by xbrianh**
This operation
  1. Verifies that file metadata is equivalent on all replicas (where file metadata is referenced by key=`files/{uuid}`
  2. sets the blob content-type equal to the file metadata content-type

### CODE REVIEW

File: dss/operations/storage.py [hunk 1]
```diff
@@ -108,6 +108,26 @@ def process_key(self, key):
                                       file_metadata_content_type=file_metadata['content-type'],
                                       blob_content_type=blob_content_type))
 
+@storage.action("repair-file-blob-metadata",
+                arguments={"--entity-type": dict(default="file", choices=["file"])})
```
Comment (DailyDreaming) [inline dss/operations/storage.py]: What other choices other than file will we need to account for in the future?

File: dss/operations/storage.py [hunk 2]
```diff
@@ -108,6 +108,26 @@ def process_key(self, key):
...
+class repair_blob_content_type(StorageOperationHandler):
+    """
+    Make blob metadata consistent with file metadata.
+    """
+    def process_key(self, key):
```
Comment (DailyDreaming) [inline dss/operations/storage.py]: Should there be a function to crawl through the entire bucket and make repairs?

File: dss/operations/storage.py (hunk 1 above)
Comment (xbrianh) [reply dss/operations/storage.py]: None, this line is there to restrict the options of `--entity-type` to just `file`.

File: dss/operations/storage.py (hunk 2 above)
Comment (xbrianh) [reply dss/operations/storage.py]: The operator should decide what to do based on the output of `scripts/dss-ops.py storage verify-file-blob-metadata`.

Once a list of repair candidates is generated, scripting may be used to either call the repair commands locally or SQS forward them for Lambda execution:
```
for key in $keys; do
  scripts/dss-ops.py storage repair-file-blob-metadata --replica $replica --keys $key --forward-to-lambda
done
```

Comment (DailyDreaming) [reply dss/operations/storage.py]: If you use the command above on only one replica as it seems to be doing, will the changes sync?

File: dss/operations/storage.py [hunk 3]
```diff
@@ -124,6 +124,26 @@ def process_key(self, key):
...
+        try:
```
Comment (Bento007) [inline dss/operations/storage.py]: Is this code run anywhere in the tests or an integration test?

**Pull request merged**